common/changes/@aws/workbench-core-authentication/test-create-user-integ-tests_2022-11-07-22-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@aws/workbench-core-authentication",
+      "comment": "BREAKING CHANGE: User interface now uses `id` instead of `uid`",
+      "type": "major"
```

Comment:
we don't have any major versions released yet, so I am wondering if this should be considered major

Reply:
Since we'll be updating the major version on GA, I dont see the harm in being explicit here.